test/test_macros.h

```diff
@@ -6,14 +6,28 @@
 #include <gsl/span>
 
 #define EXPECT_THROW_MSG(TRY_BLOCK, EXCEPTION_TYPE, MESSAGE)                   \
-  EXPECT_THROW({                                                               \
-    try {                                                                      \
-      TRY_BLOCK;                                                               \
-    } catch (const EXCEPTION_TYPE &e) {                                        \
-      EXPECT_STREQ(MESSAGE, e.what());                                         \
-      throw;                                                                   \
-    }                                                                          \
-  }, EXCEPTION_TYPE);
+  EXPECT_THROW(                                                                \
+      {                                                                        \
```

Comment:
Is your `clang-format` giving different result here than mine?

Reply:
Yes I think so. We should put a `.clang-format` file into the repository root.